src/index.ts

```diff
@@ -142,6 +153,18 @@ async function main() {
 				.conflicts(['rounds, round, block'])
 				.argParser(cliParseInt)
 		)
+		.addOption(
+			new Option(
+				'-g, --gas',
```

Comment:
I'd suggest changing this to something like `--show-gas` as this seems to be more self-explanatory